ci_framework/roles/artifacts/tasks/cluster_info.yml

```diff
@@ -30,13 +30,35 @@
     - name: Dump the logs of all pods
       ansible.builtin.shell: |
         all_pods=$(oc get pods -o name)
+        # this is implicitly relying on the fact that the pod name
+        # prefixed with "pod/" so that the redirect places the logs
+        # in the correct pod directory.
         mkdir pod
-        for pod in $all_pods; do
-          echo $pod
-          oc logs $pod > ${pod}-logs.txt
-          oc get -o yaml $pod > ${pod}.yaml
-          oc describe $pod > ${pod}-describe.txt
+        for pod in ${all_pods}; do
+          echo ${pod}
+          oc logs ${pod} > ${pod}-logs.txt
+          oc get -o yaml ${pod} > ${pod}.yaml
+          oc describe ${pod} > ${pod}-describe.txt
         done
+        # so we rename the directory to "pods" to consistently
+        # use plural for directories.
+        mv pod pods
+      args:
+        chdir: "{{ cifmw_artifacts_basedir }}/logs/edpm"
+      changed_when: true
+      ignore_errors: true
+
+    - name: Dump all openstack CRs
+      ansible.builtin.shell: |
+        mkdir crs
+        pushd crs
+          all_crds=$(oc get crd | grep openstack | awk '{print $1}' | awk -F '.' '{print $1}')
+          for cr in ${all_crds}; do
+            echo ${crd}
+            oc get -o yaml ${cr} > ${cr}.yaml
+            oc describe ${cr} > ${cr}-describe.txt
+          done
+        popd
       args:
         chdir: "{{ cifmw_artifacts_basedir }}/logs/edpm"
       changed_when: true
```

Comment:
why don't you have the `ignore_errors: true` here? Just wanting to ensure we're consistent and doing the right thing :).

Reply:
actully looking locally  i do it just did not show the last line in the file by default and you have to click...

have i said that github sucks for code review lately :)

Reply:
dang....